skycatalogs/scripts/create_sc.py

```diff
@@ -57,10 +58,23 @@
                      Persist git provenance information for each file
                      written. Only supported format currently is as a
                      small yaml file, written to the data directory.''')
-
 parser.add_argument('--dc2', action='store_true', help='If supplied provide values comparable to those used for DC2 run. Default is False')
+parser.add_argument('--file', default='', help='''
+                    path to yaml file associating option names with values. Values for any options
+                    included will take precedence.''')
 
 args = parser.parse_args()
+
+if len(args.file) > 0:
```

Comment:
I think
```
if args.file:
```
would be more idomatic (and you could also use `None` as the default).